include/vulkan/layer/vk_layer_settings.h

```diff
@@ -21,44 +21,178 @@
 
 #pragma once
 
-#include <string>
-#include <vector>
-
 #include <vulkan/vk_layer.h>
 #include <vulkan/vulkan.h>
 
-namespace vku {
+// VK_EXT_layer_settings
```

Comment:
Ensure the interface is C friendly.

```suggestion
#ifdef __cplusplus
extern "C" {


...


#ifdef __cplusplus
}
#endif
// VK_EXT_layer_settings
```